README.md

```diff
@@ -13,21 +11,34 @@ Kouvola city needed a new channel to reach Kouvola municipality inhabitants.
     * [React-native-inappbrowser-reborn](https://github.com/proyecto26/react-native-inappbrowser) in web views which require navigation functionality
     * Redirect to open other Kouvola related applications or redirecting users to store pages of these applications
 
+Modules which are managed by City of Kouvola used inside the app:
+* [open-city-kvl-modules](https://github.com/City-of-Kouvola/open-city-kvl-modules) repository, which contains modules that implement some specific functionality and can be installed to a derivate app.
+* [Library_card_module](https://github.com/City-of-Kouvola/Library_card_module) repository to bring library card to the device. Library module creates bar code into the application screen
+
 
 Kouvola City App has been developed using:
 
-`"React": "16.9.0"`
+`"React": "18.1.0"`
```

Comment:
Change this version number to 18.2.0,  since it is the version being used at this moment according to package.json